src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import { Navbar, Footer } from "./components/layouts";
+import { Home, About, Van, VanDetail } from "./pages";
+
+const App = () => {
+  return (
+    <div className="min-h-screen bg-[#FFF7ED] relative">
+      <BrowserRouter>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/about" element={<About />} />
+          <Route path="/van" element={<Van />} />
+          <Route path="/van/:id" element={<VanDetail />} />
```

Comment:
The path name `van` might be confusing as they are multiple vans on the page, so `vans` would be a better path name.